react/toast/ToastItem.tsx

```diff
@@ -43,7 +47,7 @@ const ToastItem = ({ variant, text, dismissible = false, id, duration }: ToastPr
     <ExpandTransition show={showItem}>
       <div className={toastClasses} role={role} id={id}>
         <span className="w-toast__icon">{icon}</span>
-        <p>{text}</p>
+        {insertText && <p>{text}</p>}
```

Comment:
Maybe just have `<p>{text}</p>` without involving the state? Since the toast lasts for a short time, a re-render with state change seems to be an overkill

Reply:
Hmm. So something is weird if a re-render makes this work.